File: Python/q1.py
def check_multiple(first, second, fromNum, toNum):
    sum = 0
    for i in range(fromNum, toNum):
        if(i % 3 == 0 or i % 5 == 0):
            sum += i
    return sum

first = (int(input("Enter the first number: ")))
second = (int(input("Entee the second number: ")))

fromNum = int(input("Enter the range from: "))
toNum = int(input("Enter the range to: "))

result = check_multiple(first, second, fromNum, toNum)
print(result)

File: Dart/q1.dart
import 'dart:io';

int multiple_Of_Tow(int first, int second, int from, int to) {
  int sum = 0;
  for (int i = from; i < to; i++) {
    if (i % first == 0 || i % second == 0) {
      sum += i;
    }
  }
  return sum;
}

void main() {
  print("Enter the two number that you want to check Multiple of it.");
  print("First Number");
  int firstNumber = int.parse(stdin.readLineSync()!);
  print("Second Number");
  int secondNumber = int.parse(stdin.readLineSync()!);
  print("Enter the range that you want to check.");
  print("From");
  int fromeRange = int.parse(stdin.readLineSync()!);
  print("To");
  int toRange = int.parse(stdin.readLineSync()!);

  int result_is =
      multiple_Of_Tow(firstNumber, secondNumber, fromeRange, toRange);
  print("The sum is: $result_is");
}
